presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -900,6 +901,33 @@ public static double chiSquaredCdf(
         return distribution.cumulativeProbability(value);
     }
 
+    @Description("inverse of Laplace cdf given mean, scale parameters and probability")
+    @ScalarFunction
+    @SqlType(StandardTypes.DOUBLE)
+    public static double inverseLaplaceCdf(
+            @SqlType(StandardTypes.DOUBLE) double mean,
+            @SqlType(StandardTypes.DOUBLE) double scale,
+            @SqlType(StandardTypes.DOUBLE) double p)
+    {
+        checkCondition(scale > 0, INVALID_FUNCTION_ARGUMENT, "scale must be greater than 0");
+        checkCondition(p >= 0 && p <= 1, INVALID_FUNCTION_ARGUMENT, "p must be in the interval [0, 1]");
+        LaplaceDistribution distribution = new LaplaceDistribution(null, mean, scale);
+        return distribution.inverseCumulativeProbability(p);
+    }
+
+    @Description("Laplace cdf given mean, scale parameters and value")
+    @ScalarFunction
+    @SqlType(StandardTypes.DOUBLE)
+    public static double laplaceCdf(
+            @SqlType(StandardTypes.DOUBLE) double mean,
+            @SqlType(StandardTypes.DOUBLE) double scale,
+            @SqlType(StandardTypes.DOUBLE) double value)
+    {
+        checkCondition(scale > 0, INVALID_FUNCTION_ARGUMENT, "scale must be greater than 0");
+        LaplaceDistribution distribution = new LaplaceDistribution(null, mean, scale);
```

Comment:
There seems to be a constructor which takes in just mean and scale.
You can use this instead. `new LaplaceDistribution(mean, scale);`

Reply:
Thanks! While that does look cleaner, passing the explicit `null` here is in line with how we handle the other distributions in `MathFunctions`. According [to the docs](https://commons.apache.org/proper/commons-math/javadocs/api-3.6.1/org/apache/commons/math3/distribution/LaplaceDistribution.html#LaplaceDistribution(double,%20double)), the `LaplaceDistribution(double, double)` constructor implicitly creates a new instance of an RNG, and they recommend explicitly passing a `null` to the `LaplaceDistribution(RandomGenerator, double, double)` constructor instead if no sampling is required (which is the case here).

Let me know if we should keep it this way or if you'd like me to change this.